.github/workflows/ci-cd.yml

```
@@ -0,0 +1,350 @@
+  # ==========================================
+  # Notifications
+  # ==========================================
+  notify-success:
+    name: Notify - Success
+    runs-on: ubuntu-latest
+    needs: [deploy]
+    if: success()
+
+    steps:
+      - name: Notify Slack - Success
+        if: ${{ secrets.SLACK_WEBHOOK_URL != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)